helm_deploy/values-preprod.yaml

```diff
@@ -42,6 +42,7 @@ generic-service:
     SEND_LEGAL_MAIL_URL: https://check-rule39-mail-preprod.prison.service.justice.gov.uk
     WELCOME_PEOPLE_INTO_PRISON_URL: https://welcome-preprod.prison.service.justice.gov.uk
     MERCURY_SUBMIT_URL: https://submit-a-mercury-report-preprod.hmpps.service.justice.gov.uk
+    IMS_SUBMIT_URL: https://manage-intelligence-preprod.prison.service.justice.gov.uk
```

Comment:
Can we change this to MANAGE_INTELLIGENCE_URL

Reply:
changed to MANAGE_INTELLIGENCE_URL